tests/integ/scala/test_dataframe_copy_into.py

```diff
@@ -1219,62 +1219,3 @@ def test_copy_into_table_non_csv_using_options(session, tmp_stage_name1):
         Utils.check_answer(session.table(table_name), assert_data * 2)
     finally:
         Utils.drop_table(session, table_name)
-
-
-def test_copy_into_table_names(session, db_parameters, tmp_stage_name1):
-    schema = session.get_current_schema()
-    test_file_on_stage = f"@{tmp_stage_name1}/{test_file_csv}"
-
-    def create_and_append_check_answer(table_name, full_table_name=None):
-        try:
-            assert session._table_exists(table_name) is False
-            Utils.create_table(
-                session, full_table_name or table_name, "a Int, b String, c Double"
-            )
-            assert session._table_exists(table_name) is True
-            assert session.table(table_name).count() == 0
-
-            df = session.read.schema(user_schema).csv(test_file_on_stage)
-            df.copy_into_table(table_name)
-            Utils.check_answer(
-                session.table(table_name),
-                [Row(1, "one", 1.2), Row(2, "two", 2.2)],
-            )
-        finally:
-            session.sql(f"drop table if exists {full_table_name or table_name}")
```

Comment:
This drop might not be working as expected. I see many temp tables in sfctest that have 3 dots in name `...`. When this test is re-added, let's make sure we drop the table name we created. Let me know if I am wrong here tho. This is the first test I found which is adding 3 dots in table name